MonoGame.Framework/Graphics/Texture2D.cs

```diff
@@ -93,6 +97,7 @@ protected Texture2D(GraphicsDevice graphicsDevice, int width, int height, bool m
             this.GraphicsDevice = graphicsDevice;
             this.width = width;
             this.height = height;
+            this._texelSize = new Vector2(1f / (float)width, 1f / (float)height);
```

Comment:
@Vodacek, 
As for making TexelSize public, what's the need for it? 
How will it work with 'ResizedToPowerOf2' textures?
Is `Vector2` (.X/.Y) good enough for a public API or should we introduce a SizeF structure (.Width/.Height)?